integration_tests/conftest.py

```diff
@@ -16,16 +16,23 @@ def sim() -> InteractiveContext:
     return simulation
 
 
-TIME_STEPS_TO_TEST = [0, 1, 10]
+TIME_STEPS_TO_TEST = [0, 1]
```

Comment:
I don't think we want this.

Reply:
Good catch I have reverted this after testing.